downstream/modules/eda/ref-performance-troubleshooting.adoc

```diff
@@ -0,0 +1,20 @@
+[id="performance-troubleshooting"]
+
+=  Performance Troubleshooting for {EDAcontroller}
+
+[role="_abstract"]
+Based on the default parameters within {EDAcontroller}, you might encounter scenarios that pose challenges to completing your workload. The following section provides descriptions of these scenarios and troubleshooting guidance.
+
+* My activation status displays as “running”, but it is not processing the events.
+** Ensure that you are using the correct event source in the rulebook activation. If the event you are expecting is coming from a source other than what is in the rulebook, {EDAcontroller} will not process the event.
+
+* My activation status displays as “running”, and {EDAcontroller} is also receiving the events, but no actions are occuring.
+** Ensure that you have set the correct conditions for matching the event and taking actions in the rulebook activation.
+
+* My activation keeps restarting in an infinite loop.
+** By default, the reset policy for rulebook activations is set to “on failure”. The restart policy can be changed by using the following procedure:
+
+***Procedure
```

Comment:
```suggestion
```

Reply:
The asciidoc renders weird here. I think you can eliminate "procedure", as the preceding sentence introduces it.